l3kernel/testfiles/m3expl001.ptex.tlg

```diff
@@ -170,6 +170,12 @@ Defining \__pdf_backend_link_minima: on line ...
 Defining \__pdf_backend_link_outerbox:n on line ...
 Defining \__pdf_backend_link_sf_save: on line ...
 Defining \__pdf_backend_link_sf_restore: on line ...
+Defining \__hook cmd/@makecol/before on line ...
+Defining \g__hook_cmd/@makecol/before_code_prop on line ...
+Defining \__hook_toplevel cmd/@makecol/before on line ...
+Defining \__hook_next cmd/@makecol/before on line ...
+Defining \g__hook_cmd/@makecol/before_labels_clist on line ...
+Defining \g__hook_cmd/@makecol/before_reversed_tl on line ...
```

Comment:
is there a way to actually test the functionality? Looks like all the tests do is showing that the hook got initialized

Reply:
A test would need to be at the PDF level: are we happy with that?

Reply:
dunno, but obviously the code is not working and the tests aren't in any way testing that it works. If we can't test it then the current tests could goo too I guess. If we can test "something" then we should do it :-)

Reply:
@FrankMittelbach How do you know that "the code is not working"?
Did you test it somehow:-)?

Reply:
just from your description of what it should do ... and very clearly doesn't

Reply:
Or it does? ... at least when
 @u-fischer works good magic on it!

Reply:
@car222222 no. Ulrike looked at it *after* Joseph altered the right box :-)

Reply:
@FrankMittelbach  Nope, it works with the wrong box provided the link doesn't break between columns :)

Reply:
LOL

Reply:
I see!  Or rather I do not as the timing seems all wrong: maybe @josephwright used his time machine to change boxes?  
Or the box is itself a time-machine -- so lease call it \__Tardis_box ??

Reply:
if you look above to my other comment on the code, it says "outdated" and that is due to Joseph changing from `\l_shipout_box` to 255 by now (we don't get email notifications on l3kernel changes) so no time machine involved. Changing `\l_shipout_box`at this point didn't do anything good, because it isn't set up yet by the OR. This is what made me say the code can't work the way it is.